http/server.go

```diff
@@ -13,6 +13,12 @@ type (
 		Use(func(http.Handler) http.Handler)
 	}
 
+	// Mounter is the interface for servers that allow mounting their endpoints
+	// into a mutex.
```

Comment:
Should that read `muxer` instead of `mutex`?

Reply:
It certainly should have been `muxer`. I'll blame my internal autocorrect 😅

```suggestion
	// into a muxer.
```